package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "serve": "ng serve",
```

Comment:
I can see why you prefer to call this serve instead of start. 
However, this is a standard script for npm and it might be confusing that this script is missing (for example, if another student is more familiar with javascript)

Are there other reasons why you suggest to change the name of the task that I'm overlooking?

Reply:
@manuwei I prefer `serve` as it is the same like used with the Angular CLI and may avoids confusion. And I changed it cause, the documentation mentioned `ng serve` to run the project. But I do not care, we can keep `start` here, if you prefer this.

Reply:
Okay, I'd prefer to keep the standard task, npm/yarn 'start' instead of serve. 
Using `ng serve` still works if we keep calling the task 'start', so I don't think it's a problem to keep the standard naming conventions.

Reply:
@manuwei done